src/api/lib/tasks/dev.rake

```diff
@@ -78,40 +78,56 @@ namespace :dev do
     Rake::Task['db:structure:verify_no_bigint'].invoke
   end
   namespace :lint do
-    namespace :rubocop do
-      require 'rubocop/rake_task'
+    unless Rails.env.production?
```

Comment:
:thought_balloon: phew, Gemfile groups baked into code. I don't know guys...

Reply:
sure

Reply:
@hennevogel what was the problem with this? we have at the moment rake task defined which uses gems that are not installed in production. So, in my opinion we shouldn't define the tasks as well... :thinking: